### ISSUE TITLE
Option to display the IP address used to contact destination

### ISSUE THREAD

**Issue by InAnimaTe** [reorx/httpstat#18]
It would be nice to find out the exact IP that was used to make the connection. This helps for things like debugging round-robin records etc..

Maybe something like `HTTPSTAT_SHOW_IP`

Great work on this btw, definitely my go to utility now ;)

**Comment by reorx** [comment]
I've also came up with this before, and since httpstat wraps curl with minimum restriction, I think it would be easy to get the IP address and port showing by add `-v` option, like this:

```bash
$ httpstat google.com -v
* Rebuilt URL to: google.com/
*   Trying 216.58.221.238...
* Connected to google.com (216.58.221.238) port 80 (#0)
...
```

But you are right, IP is one of the most important information we need to know about a HTTP request, besides, `-v` seems a bit tedious. I'm going to add IP display soon, with `HTTPSTAT_SHOW_IP=true` by default :)

**Comment by reorx** [comment]
@InAnimaTe BTY, what do you think of local IP/port when requesting, is it also necessary to show with remote IP/port ?

**Comment by InAnimaTe** [comment]
Yeah actually local IP/Port used would also be really useful. This way, I know exactly how the packet left me without having to fire up wireshark (but I still could if I wanted more information). For this information, maybe a `HTTPSTAT_SHOW_LOCAL` variable of some sort.

Glad you like the idea! Can't wait to see it implemented!

**Comment by reorx** [comment]
I have made an implementation on branch [show-ip](https://github.com/reorx/httpstat/tree/show-ip) , how do you think of it?

**Comment by reorx** [comment]
Implemented and merged in #20

**Comment by InAnimaTe** [comment]
@reorx sorry for not getting back to you before. This looks absolutely wonderful! Glad its now the default!
